types/twilio-video/index.d.ts

```diff
@@ -569,8 +574,9 @@ export interface LocalDataTrackOptions {
 }
 export type LocalTrack = LocalAudioTrack | LocalVideoTrack | LocalDataTrack;
 export interface LocalTrackOptions {
-    logLevel: LogLevel | LogLevels;
+    logLevel?: LogLevel | LogLevels;
     name?: string;
+    priority?: TrackPriority;
```

Comment:
Can we make a new interface called `LocalTrackPublishOptions` and add `priority` into it?
In twilio docs, they are distinguished.

https://media.twiliocdn.com/sdk/js/video/releases/2.3.0/docs/global.html#LocalTrackPublishOptions